20190519-WebAbstract/BDA_FS19_WebAbstract.tex

```diff
@@ -108,11 +122,24 @@ \section{Ergebnisse}
 	\includegraphics[keepaspectratio,width=2.7cm]{img/HSLU_Logo_Header}
 \end{textblock*}
 
-\section{Lösungskonzept}
+\section{Ergebnisse}
+Wie in der Aufgabenstellung beschrieben wurden zwei Konzepte ausgearbeitet. Das eine konzentrierte sich auf das automatische Auffinden von Exemplaren im Hochregallager selbst, während das andere eine Station voraussieht, welche fehlplatzierte Exemplare vor dem Einlagern identifiert und den betreffenden Behälter ausschleust (in Abbildung \ref{fig:PosAntennen} sind die Antennenposition dieser Station dargestellt).
```

Comment:
Das eine konzentrierte sich auf das  -> Währen das erste sich auf das automatische Auffinden von Exemplaren im Hochregallager selber konzentriert, befasst sich das zweite Konzept mit der Entwicklung einer Kontrollstation, welches fehlplatzierte Exemplare vor der Einlagerung identifizieren kann und den betreffenden Behälter in den dafür vorgesehenen ausschleuseplatz ausschleust (in ...).